pkg/controller/build/build_controller.go

```diff
@@ -97,6 +100,17 @@ const (
 	osImageURLConfigKey = "osImageURL"
 )
 
+const (
+	// Name of the etc-pki-entitlement secret from the openshift-config-managed namespace.
```

Comment:
note: In the future, we may not want to hard-code these values. They could potentially become another set of fields on the MachineOSConfig object as well as something the cluster admin would provide.